tmpl/index.twig

```diff
@@ -0,0 +1,148 @@
+                        <table class="table text-white-50">
+                            <thead>
+                            <tr>
+                                <th class="text-left border-0">Title</th>
+                                <th class="border-0 text-center">Comb</th>
+                                <th class="border-0 text-center">Pres</th>
+                                <th class="border-0 text-center">Cam</th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                            {% for livestream in livestreams %}
+                                <tr>
+                                    <td class="text-left tr-title">{{ livestream.info.name }}</td>
+                                    <td class="align-middle text-center">
+                                        {% if livestream.links.comb is defined %} {# check if recordings exist. Some lectures may only have pres #}
+                                            <a href="watch/{{ livestream.links.comb }}">
+                                                <i class="fa fa-th-large"></i></a>
+                                        {% endif %}
+                                    </td>
+                                    <td class="align-middle text-center">
+                                        {% if livestream.links.pres is defined %} {# check if recordings exist. Some lectures may only have pres #}
+                                            <a href="watch/{{ livestream.links.pres }}">
+                                                <i class="fa fa-pencil-square-o"></i></a>
+                                        {% endif %}
+                                    </td>
+                                    <td class="align-middle text-center">
+                                        {% if livestream.links.cam is defined %} {# check if recordings exist. Some lectures may only have pres #}
+                                            <a href="watch/{{ livestream.links.cam }}">
+                                                <i class="fa fa-video-camera"></i></a>
+                                        {% endif %}
+                                    </td>
+                                </tr>
+                            {% endfor %}
+                            </tbody>
+                        </table>
+                    </div>
+                {% endif %}
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-8 offset-md-2">
+                <h3 class="subtitle">Video On Demand<br></h3>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col col-md-8 offset-md-2 p-0">
+                <div role="tablist" id="accordion-1">
+                    {% for vodElem in vod %}
+                        <div class="card {% if loop.index < loop.length %} border-bottom-0 {% endif %} ">
```

Comment:
nit: This would be cleaner using a `:last-child` CSS selector.

Reply:
Cool! Wasn't aware of that.